pom.xml

```diff
@@ -86,7 +87,7 @@
         <junit.version>4.13.2</junit.version>
         <log4j.version>1.2.17</log4j.version>
         <shrinkwrap.version>1.2.6</shrinkwrap.version>
-        <shrinkwrap.descriptors.version>1.1.0-beta-1</shrinkwrap.descriptors.version>
+        <shrinkwrap.descriptors.version>2.0.0</shrinkwrap.descriptors.version>
```

Comment:
I think even this version is still not jakarta-friendly (and the project is pretty much dead). But that's something we will need to eventually solve on the TCK level.

Reply:
2.0.0 is the current version of the TCK, but if it needs to be removed from the TCK I can do that.

Reply:
We have no replacement ATM. Even if it creates outdates descriptors (w/o jakarta namespace) it still works.
The only other way is to go and create all the descriptions manually under TCK which I don't think is worth it.
So, let's just keep it this way for now.